started refining network config chapter (#641)
openstack-k8s-operators/data-plane-adoption: docs_user/assemblies/assembly_configuring-network-for-RHOSO-deployment.adoc

:context: configuring-network

= Configuring the network for the {rhos_long_noacro} deployment

When you adopt a new {rhos_long} deployment, you must align the network
configuration with the adopted cluster to maintain connectivity for existing
workloads.

> You need to incorporate the existing network configuration in the following steps:

Reviewer: ```suggestion
Perform the following tasks to incorporate the existing network configuration:
```
Suggestion to avoid using "steps" here cos it isn't a procedure, sounds more like a workflow.

* You configure {rhocp_long} worker nodes to align VLAN tags and IP Address Management (IPAM) configuration with the existing deployment.
* You configure control plane services to use compatible IP ranges for service and load-balancing IP addresses.
* You configure data plane nodes to use corresponding compatible configuration for VLAN tags and IPAM.

> * You can either reuse IPAM configuration from the existing deployment or, depending on IP address availability in the existing allocation pools, define new ranges to be used for the new control plane services. If you define new ranges, you configure IP routing between the old and new ranges. For more information, see xref:planning-your-ipam-configuration_configuring-network[Planning your IPAM configuration].

Reviewer: ```suggestion
* Reuse IPAM configuration from the existing deployment or, depending on IP address availability in the existing allocation pools, define new ranges for the new control plane services. If you define new ranges, you configure IP routing between the old and new ranges. For more information, see xref:planning-your-ipam-configuration_configuring-network[Planning your IPAM configuration].
```
